release/models/gnsi/openconfig-gnsi-credentialz.yang

```diff
@@ -232,9 +258,16 @@ module openconfig-gnsi-credentialz {
           "Console-related state.";
 
         uses counters;
+
+        leaf glome {
+          type boolean;
+          description
+            "Whether GLOME is configured or not.";
```

Comment:
Can you define what configured means?  Do you mean if glome is enabled?  If yes, in OC it's most common to have an 'enabled' leaf.

Reply:
Done, changed to enabled.